### PR TITLE
Removed uneeded dependency on symfony/finder

### PR DESCRIPTION
**Pull request by Nyholm**
We do never use it.

### CODE REVIEW

**Comment by xabbuh** [comment]
The [use statement in the Updater class](https://github.com/padraic/phar-updater/blob/master/src/Updater.php#L23) should be removed too.

File: src/Updater.php [hunk 1]
```diff
@@ -20,7 +20,6 @@
 use Humbug\SelfUpdate\Strategy\StrategyInterface;
 use Humbug\SelfUpdate\Strategy\ShaStrategy;
 use Humbug\SelfUpdate\Strategy\GithubStrategy;
-use Symfony\Component\Finder\Finder;
```
Comment (Nyholm) [inline src/Updater.php]: The use statement is already removed.

**Comment by sagikazarmark** [comment]
:+1:

File: src/Updater.php (hunk 1 above)
Comment (xabbuh) [inline src/Updater.php]: oops :(

**Comment by padraic** [comment]
Merging...merging...no explosions...merging ;) Thanks!

**Comment by webmozart** [comment]
@padraic Thanks a lot! :)

**Comment by Nyholm** [comment]
Thank you for merging! Could you tag 1.0.3?

**Comment by padraic** [comment]
I'll be tagging shortly @Nyholm

**Pull request merged**